CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 23.1.1-RC1
+Besu 23.1.1-RC is only a required update only Sepolia users this is optional for Mainnet users.
```

Comment:
nit: Do we even want to say it's optional for mainnet? I'd say it's discouraged if anything since we haven't burned it in yet! This is less important than the release notes, hence the nit.

Reply:
That message is broken. Will fix it up once get the DCO sorted

Reply:
Updated to "Besu 23.1.1-RC is a required update only for users who want to test Shanghai on Sepolia."